charts/platform-service/values.yaml

```diff
@@ -83,8 +83,8 @@ defaultRouting:
   enabled: true
 
   kubeitDefaultHosts:
-    tenantPrefixEnabled: true
     enabled: false
+    tenantPrefixEnabled: true

```

Comment:
any change here?

Reply:
tenantPrefixEnabled required the enabled flag so it makes more sense this way... also it's alphabetical order :-)